config/dispatcher/dispatcher.ex

```diff
@@ -0,0 +1,36 @@
+efmodule Dispatcher do
+  use Matcher
+  define_accept_types [
+    html: [ "text/html", "application/xhtml+html" ],
+    json: [ "application/json", "application/vnd.api+json" ]
+  ]
+
+  @any %{}
+  @json %{ accept: %{ json: true } }
+  @html %{ accept: %{ html: true } }
+
+  define_layers [ :static, :web_page, :services, :fall_back, :not_found ]
+
+  ###############
+  # STATIC
+  ###############
+  get "/assets/*path", %{ layer: :static } do
+    Proxy.forward conn, path, "http://frontend/assets/"
```

Comment:
I suppose rules are copied from elsewhere, but try to be consistent in this file. Either use `Proxy.forward` or `forward` but not a mix. It adds confusion/complexity where it shouldn't.